testing/scripts/test_helpdesk.py

```
@@ -691,8 +691,55 @@ async def admin_update_ticket(post_id, status=None, priority=None,
-        # ── [15] Cleanup ──────────────────────────────────────────────────────────
-        print("\n[15] Cleanup")
+        # ── [15] Plugin Icons ─────────────────────────────────────────────────────
+        print("\n[15] Plugin Icons")
+
+        CDN_BASE     = "https://media.seanmousseau.com/file/seanmousseau/assets/logos/swh"
+        CDN_ICON_128 = f"{CDN_BASE}/icon-128x128.png"
+        CDN_ICON_256 = f"{CDN_BASE}/icon-256x256.png"
+
+        # CDN reachability — use curl (hardcoded URLs, avoids dynamic urllib warning)
+        for cdn_url, label in ((CDN_ICON_128, "1x"), (CDN_ICON_256, "2x")):
+            result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)